interop-testing/src/main/java/io/grpc/testing/integration/AbstractInteropTest.java

```diff
@@ -1254,7 +1254,7 @@ protected void assertRemoteAddr(String expectedRemoteAddress) {
 
     HostAndPort remoteAddress = HostAndPort.fromString(serverCallCapture.get().getAttributes()
             .get(Grpc.TRANSPORT_ATTR_REMOTE_ADDR).toString());
-    assertEquals(expectedRemoteAddress, remoteAddress.getHost());
+    assertEquals(expectedRemoteAddress, remoteAddress.getHostText());
```

Comment:
This is deprecated, and will make it harder to go back up a version.   Do we even need to use it?